eng/common/scripts/Verify-Resource-Ref.ps1

```diff
@@ -1,6 +1,6 @@
 . (Join-Path $PSScriptRoot common.ps1)
 Install-Module -Name powershell-yaml -RequiredVersion 0.4.1 -Force -Scope CurrentUser
-$ymlfiles = Get-ChildItem $RepoRoot -recurse | Where-Object {$_ -like '*.yml'}
+$ymlfiles = Get-ChildItem $RepoRoot -recurse | Where-Object {$_ -like '*.yml' -and $_ -notlike '*\invalid\*'}
```

Comment:
I wonder if we should instead just handle the error below when parsing the yml file and if it fails we will output a skipping message and go on.

Reply:
Hey @mccoyp @chidozieononiwu executed on Wes's comment above. 

[See PR here](https://github.com/Azure/azure-sdk-tools/pull/3568) We shouldn't be getting aggregate report failures from this anymore.

Reply:
Nah McCoy, I definitely should be watching my notifications and followed up for you. Thanks for making the effort to propose a fix!